Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java

```diff
@@ -287,4 +293,8 @@ private Date convertXMLGregorianCalendarToDate(XMLGregorianCalendar xmlCalDate)
         LOG.info("eventDate -> " + eventDate);
         return eventDate;
     }
+
+    private String getRelatesTo(AssertionType assertion) {
+        return NullChecker.isNotNullish(assertion.getRelatesToList()) ? assertion.getRelatesToList().get(0) : null;
```

Comment:
To be consistent - Can we set relatesTo in LogEventRequestType in AuditTransforms?